### PR TITLE
Link sqlflow container to sqlflowdata container

### PR DESCRIPTION
**Pull request by farmercode**
fix network connect problem,link sqlflow container to sqlflowdata container.

### CODE REVIEW

File: doc/installation.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ After data is popularized in MySQL, let's test the installation from running a q
    using Docker for Linux, please change `host.docker.internal:3306` to `localhost:3306`.
 
    ```
-   docker run --rm -it -p 8888:8888 sqlflow/sqlflow:latest \
-   bash -c "sqlflowserver --datasource='mysql://root:root@tcp(host.docker.internal:3306)/?maxAllowedPacket=0' &
+   docker run --rm -it -p 8888:8888 --link sqlflowdata:data_server sqlflow/sqlflow:latest \
```
Comment (hungry1526) [inline doc/installation.md:26]: This PR conflicts PR #353, please take a look at that PR first.

Comment (farmercode) [reply doc/installation.md:26]: ok,I will check it!

**Comment by tonyyang-svail** [comment]
@farmercode Thanks for updating the documentation. May I ask on which OS are you using?

**Comment by farmercode** [comment]
@tonyyang-svail I am using macOS 10.14!

File: doc/installation.md (hunk 1 above)
Comment (farmercode) [reply doc/installation.md:26]: @hungry1526 If use PR#353, my PR will be nothing! It just fix the old version network problem of containers. So best is I cancel it!

Comment (hungry1526) [reply doc/installation.md:26]: You are very welcome, we encourage PR like this, please keep doing it!

**Comment by farmercode** [comment]
install.md has changed!

**Pull request closed without merging**